mailpoet/assets/css/src/mailpoet-templates.scss

```diff
@@ -59,15 +59,30 @@
 
 .mailpoet-templates-card {
   align-items: flex-end;
+  border: 1px solid white;
+  cursor: pointer;
   display: flex;
   flex-direction: column;
   flex-shrink: 0;
-  width: 336px;
+  width: 334px;
 
   > div {
     display: grid;
     grid-template-rows: auto 1fr;
   }
+
+  &:hover {
+    background: #fff;
+    border: 1px solid #dcdcde;
+    box-shadow: 0 3px 6px rgba(0, 0, 0, .15);
+    color: inherit;
+  }
```

Comment:
I'm not sure how this happens, but with the hover state, the border seems to jump 1px inside and get a different radius.

At the same time, I just noticed, that while the cards react to a click, they don't seem to be accessible (= they are not buttons with `tabindex` and stuff). I suppose this is rather OK, because TAB selects the headings (but it also likely means a screen reader wouldn't read the subtitle when selected). If it's a component we should reuse across the plugin, we could consider creating a ticket to improve this.

Reply:
> At the same time, I just noticed, that while the cards react to a click, they don't seem to be accessible (= they are not buttons with tabindex and stuff). I suppose this is rather OK, because TAB selects the headings (but it also likely means a screen reader wouldn't read the subtitle when selected). If it's a component we should reuse across the plugin, we could consider creating a ticket to improve this.

I'm using the <Card> WordPress component, but maybe the problem is that this component is not meant to be clickable and thus, this is causing the accessibility problem that you mentioned. I think that creating a separate ticket to look into this is a good approach.

Reply:
Ticket here: https://mailpoet.atlassian.net/browse/MAILPOET-5585